pom.xml

```diff
@@ -5,16 +5,16 @@
 
 	<groupId>io.apimatic</groupId>
 	<artifactId>core</artifactId>
-	<version>0.1.4</version>
+	<version>0.1.6</version>
```

Comment:
This should be patched as 0.1.5 as the last patch was 0.1.4.

Reply:
This has been replaced with version 0.1.5.